cli.js

```diff
@@ -142,17 +207,30 @@ async function measureUpload(bytes, iterations) {
   return measurements;
 }
 
+function logInfo(text, data) {
+  console.log(bold(' '.repeat(15 - text.length), `${text}:`, blue(data)));
+}
+
 function logSpeedTestResult(size, test) {
   const speed = stats.median(test).toFixed(2);
   console.log(
-    bold(' '.repeat(7 - size.length), size, 'speed:', yellow(`${speed} Mbps`))
+    bold(' '.repeat(9 - size.length), size, 'speed:', yellow(`${speed} Mbps`))
   );
 }
 
 async function speedTest() {
+  const serverLocationData = await fetchServerLocationData();
+
+  const { ip, loc, colo } = await fetchCfCdnCgiTrace();
+
+  const city = serverLocationData[colo];
+
+  logInfo('Your IP', `${ip} (${loc})`);
+  logInfo('Server location', `${city} (${colo})`);
```

Comment:
Swap `Your IP` and `Server location` so the former appears above `Latency`, which is the same length, and all numbers will be grouped together.